css-ui-4/Overview.bs

```diff
@@ -2166,6 +2166,129 @@ Document languages should specify
+        <li>
+            If |computedAppearance| is ''appearance/none'', then return ''appearance: none''.
+
+        <li>
+            Let |computedAppearance| be the [=computed value=] of |element|'s 'appearance' property.
+
+        <li>
+            Let |authorProps| be a map of all properties that have a <a>cascaded value</a> for
+            |element|, where the [=Author Origin=], [=Animation Origin=] and [=Transition Origin=]
+            [=Cascade Origins=] are included and the [=User Origin=] and [=UA Origin=] [=cascade origins=]
+            are ignored, where the key is the property name and the value is that <a>cascaded value</a>.
+
+        <li>
+            If |element| can be a ''button'', given |computedAppearance|, then return the appropriate widget with
+            |authorProps| set to |authorProps|, |expected| set to button, and |fallback| set to
+            ''appearance/none''.
+
+        <li>
+            If |element| can be a ''textfield'', given |computedAppearance|, then return the appropriate widget with
+            |authorProps| set to |authorProps|, |expected| set to textfield, and |fallback| set to
+            ''appearance/none''.
+
+        <li>
+            If |element| can be a ''searchfield'', given |computedAppearance|, then return the appropriate widget with
+            |authorProps| set to |authorProps|, |expected| set to searchfield, and |fallback| set to
+            ''appearance/none''.
+
+        <li>
+            If |element| can be a ''slider-horizontal'', given |computedAppearance|, then return slider-horizontal.
+
+        <li>
+            If |element| can be a ''menulist-button'', given |computedAppearance|, then return menulist-button.
+
+        <li>
+            If |element| can be a ''menulist'', given |computedAppearance|, then return menulist.
+
+        <li>
+            If |element| can be a ''textarea'', given |computedAppearance|, then return the appropriate widget with
+            |authorProps| set to |authorProps|, |expected| set to textarea, and |fallback| set to
+            ''appearance/none''.
+
+        <li>
+            If |element| can be a ''progress-bar'', given |computedAppearance|, then return progress-bar.
+
+        <li>
+            If |element| can be a ''meter'', given |computedAppearance|, then return meter.
+
+        <li>
+            Return ''appearance/none''.
+    </ol>
+
+    The appropriate <a>widget</a>, given a map of properties |authorProps|, and an expected
+    kind of widget |expected|, and a fallback kind of widget |fallback|, is as follows:
+
+    <ol>
+        <li>For each |prop| of the properties that disable native appearance for <a>widgets</a>:
+
+            <ol>
+                <li>If |authorProps|[|prop|] exists, and it is not ''revert'', then return |fallback|.
+            </ol>
+
+        <li>Return |expected|.
+    </ol>
+</div>
+
+The properties that disable <a>native appearance</a> for <a>widgets</a> are:
```

Comment:
`The <dfn export>properties that disable native appearance for widgets</dfn> are:`